wgengine/router/router_default.go

```diff
@@ -7,12 +7,16 @@
 package router
 
 import (
+	"fmt"
+	"runtime"
+
 	"golang.zx2c4.com/wireguard/tun"
 	"tailscale.com/types/logger"
+	"tailscale.com/wgengine/monitor"
 )
 
-func newUserspaceRouter(logf logger.Logf, tunname string, tunDev tun.Device, netChanged func()) Router {
-	return NewFakeRouter(logf, tunname, tunDev, netChanged)
+func newUserspaceRouter(logf logger.Logf, tunname string, tunDev tun.Device, linkMon *monitor.Mon) Router {
+	panic(fmt.Sprintf("unsupported OS %q", runtime.GOOS))
```

Comment:
is there a netstack-only os this is going to break?

Reply:
Turns out this code didn't even compile before. It called a function which didn't exist in the code.

But we have no GOOS for which WireGuard compiles today but we don't.

So this is a useless change that can't currently be exercised.